schemas/user.py

```diff
@@ -45,26 +45,23 @@ def validate_email(cls, v):
         message = False
 
         if not v:
-            message = "Електронний адрес не може бути порожнім"
+            message = "The email address cannot be empty."
 
         elif not re.fullmatch(regex, v):
-            message = f"Невірний формат адресу електронної пошти: {v}."
+            message = f"Invalid email address format: {v}."
 
         if message:
-            raise ValueError(message)
-
+            raise BackendException(message=message)
```

Comment:
Don't use `BackendException` inside Pydantic schemas validation. it should be handled by another handler: `validation_exception_handler`.